src/puppetlabs/pcp/message.clj

```diff
@@ -198,7 +198,7 @@
           data-frame (or (:data data-chunk) (byte-array 0))
           data-flags (or (get-in data-chunk [:descriptor :flags]) #{})]
       (try+ (s/validate Envelope envelope)
-            (catch Throwable _
+            (catch Object _
```

Comment:
Perhaps this should go in a separate maint commit.
Making the catch more general seems valid though.